src/math.js

```diff
@@ -46,5 +46,16 @@ engine.mod = function(x, y){
   return x % y;
 };
 
+engine.natlog = function(x){
+  if (x.length > 0){
```

Comment:
Is this check needed?  You did not do it for baselog.

Reply:
If parameter was not set natlog returns -Infinity, not []. But baselog returns []. It happens because ln() haven't "arity" property (fhirpath.js line 99) and I don't know why it works in that way.

Reply:
ln:             {fn: math.natlog},
 log:            {fn: math.baselog, arity:  {1: ["Number"]}, nullable: true},
It's from fhirpath.js.

Reply:
Yes, that's okay then.  (I don't know why it works that way either-- that is Niquola's code-- but as long as the output is correct, it doesn't matter.)